pyproject.toml

```diff
@@ -97,6 +97,14 @@ extended_tasks = [
   "openai", # llm as a judge using openai models
 ]
 s3 = ["s3fs"]
+multilingual = [
+    "stanza",
+    "pyvi",
+    "pythainlp",
+    "jieba",
+    "indic-nlp-library",
+    "kiwipiepy",
+]
```

Comment:
Do we need all of these additional reqs? Ideally, if some of these are only imported for a function or 2, we'd rather copy the required functions instead of pushing for so many new imports (that's what we do for some metrics for ex)